### PR TITLE
Feature webinterface redo

### LINKED ISSUE #204: Web Interface - Actions per team
As a player, I would like to have multiple types of interactions with the game, so I can choose my own way to play. 

Players should be able to interact with the game in a number of ways, such as placing bombs and mines.

This is a subtask of #203 - Web Interface

### LINKED ISSUE #206: Web Interface - Fullscreen
As developers, we would like the web interface to run in fullscreen, so it is nicer to look at and easier to work with.

This task involves the implementation of the correct HTML 5 code, to show the web interface in fullscreen.

This is a subtask of #203 - Web Interface

### LINKED ISSUE #276: Reduce WebInterface delay
The clients only send one request to the server, and get everything they need in one response.

### PR DESCRIPTION
**Pull request by Geweldig**
Adds new actions to the webinterface.
Changes the backend so we only send one request instead of 3 (#276).
Adds the fullscreen API.

Closes #204, Closes #206, Closes #276

### CODE REVIEW

File: src/main/java/nl/tudelft/contextproject/util/webinterface/ActionUtil.java [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+			case "placemine":
+				placeMine(xCoord, yCoord);
+				break;
+			case "spawnenemy":
+				spawnEnemy(xCoord, yCoord);
+				break;
+			case "dropbait":
+				dropBait(xCoord, yCoord);
+				break;
+			default:
```
Comment (ChielBruin) [inline src/main/java/nl/tudelft/contextproject/util/webinterface/ActionUtil.java:45]: Only PlaceBomb is covered by tests, you might want to test the other cases as well

Comment (Geweldig) [inline src/main/java/nl/tudelft/contextproject/util/webinterface/ActionUtil.java:45]: This entire class isn't tested as most of the functionality isn't there yet. All tests will be written once all methods have been added.

**Comment by ChielBruin** [comment]
Looks good! Can be merged in my opinion

File: src/main/java/nl/tudelft/contextproject/util/webinterface/ActionUtil.java (hunk 1 above)
Comment (Diocruel) [inline src/main/java/nl/tudelft/contextproject/util/webinterface/ActionUtil.java:45]: A bit further on there are also methodes for place Mine and place pitfall, shouldn't they be tested as they are now implemented?

Comment (Geweldig) [inline src/main/java/nl/tudelft/contextproject/util/webinterface/ActionUtil.java:45]: All tests will be written once all functionality has been implemented.

**Comment by Diocruel** [comment]
This is good to merge in my opinion.

**Pull request merged**